### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by JFer11**
In this branch we have already implemented and tested migrations functionalities.

### CODE REVIEW

File: migrations/README [hunk 1]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (joaquinco) [inline migrations/README]: What is this describing? Maybe you need to add more information. And remember the trailing new line

File: training/app.py [hunk 2]
```diff
@@ -1,21 +1,62 @@
...
+    admin = Admin(app, name='Users admin', template_mode='bootstrap3')
+    from training.controllers.admin.admin_web import UserModelView
+    from training.models.users import User
+    admin.add_view(UserModelView(User, db.session))
+
+    with app.app_context():
+        db.create_all()
+        db.session.commit()
+
+    print("Bien!")
```
Comment (joaquinco) [inline training/app.py]: Do you still need this?

File: training/background_tasks/tasks.py [hunk 3]
```diff
@@ -18,9 +19,5 @@ def example(seconds):
 
 def send_email_function(msg):
     with app.app_context():
+        # mail.suppress = True
```
Comment (joaquinco) [inline training/background_tasks/tasks.py]: can you remove the comented code?

**Comment by joaquinco** [comment]
I think you need to update the requirements.txt

**Comment by joaquinco** [comment]
Can you update the README with the migrations instructions?

File: training/app.py (hunk 2 above)
Comment (dcousin24) [reply training/app.py]: Remove that print please

Comment (joaquinco) [reply training/app.py]: I meant the db.create_all(), as I understand it's handled by flask migrations now

**Review by joaquinco — APPROVED** [review]

**Pull request merged**